<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="actiTIME - Desktop Smoke"
           tests="8"
           failures="2"
           errors="0"
           skipped="0"
           time="156.647">
  <properties />
            <testcase name="TC05_CreateAUser" time="14.655" classname="com.actitime.testscripts.TestScriptsWeb">
                <failure type="java.lang.AssertionError"
                  message="User having username: James_Nuthan already exists!"
              >
<![CDATA[java.lang.AssertionError: User having username: James_Nuthan already exists!
  org.testng.Assert.fail(Assert.java:94)
  com.actitime.webpageobjects.UsersPO.createUser(UsersPO.java:95)
  com.actitime.testscripts.TestScriptsWeb.TC05_CreateAUser(TestScriptsWeb.java:39)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  com.actitime.genericlibrary.XMLUtility.autoRunXml(XMLUtility.java:299)
  com.actitime.driver.Driver.main(Driver.java:97)
]]>
      </failure>

          </testcase>
            <testcase name="TC07_DeleteUser" time="18.120" classname="com.actitime.testscripts.TestScriptsWeb">
                <failure type="java.lang.AssertionError"
                  message="Unable to delete user"
              >
<![CDATA[java.lang.AssertionError: Unable to delete user
  org.testng.Assert.fail(Assert.java:94)
  com.actitime.webpageobjects.UsersPO.deleteUser(UsersPO.java:197)
  com.actitime.testscripts.TestScriptsWeb.TC07_DeleteUser(TestScriptsWeb.java:51)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  org.testng.TestRunner.privateRun(TestRunner.java:782)
  org.testng.TestRunner.run(TestRunner.java:632)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
  org.testng.SuiteRunner.run(SuiteRunner.java:268)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
  org.testng.TestNG.run(TestNG.java:1064)
  com.actitime.genericlibrary.XMLUtility.autoRunXml(XMLUtility.java:299)
  com.actitime.driver.Driver.main(Driver.java:97)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="TC01_SignIntoApplication" time="10.627" classname="com.actitime.testscripts.TestScriptsWeb" />
                <testcase name="TC02_NavigateToTasks" time="13.279" classname="com.actitime.testscripts.TestScriptsWeb" />
                <testcase name="TC04_NavigateToUsers" time="11.081" classname="com.actitime.testscripts.TestScriptsWeb" />
                <testcase name="TC06_VerifyExistingUser" time="67.507" classname="com.actitime.testscripts.TestScriptsWeb" />
                <testcase name="TC03_NavigateToReports" time="11.074" classname="com.actitime.testscripts.TestScriptsWeb" />
                <testcase name="TC08_SignOutOfApplication" time="10.304" classname="com.actitime.testscripts.TestScriptsWeb" />
      
</testsuite>
